src/gerbil/runtime/gambit.ss

```diff
@@ -0,0 +1,65 @@
+;;; -*- Gerbil -*-
+;;; © vyzo
+;;; Gambit builtins
+prelude: :gerbil/core
+package: gerbil/runtime
+namespace: #f
+
+(export #t)
+
+;; compatibilityf for prerelease gambit changes
+(cond-expand
+  (,(with-catch false (cut eval '##max-char))
```

Comment:
Isn't the entire check reversed? You want to define ##max-char if *not* defined yet.

Reply:
removed ##max-char altogether, it is max-char-point now

Reply:
but yes, you sre right